flask_admin/templates/bootstrap2/admin/lib.html

```diff
@@ -187,6 +187,10 @@
   {% if config.MAPBOX_MAP_ID %}
   <script>
   window.MAPBOX_MAP_ID = "{{ config.MAPBOX_MAP_ID }}";
+  {% if config.get(MAPBOX_API_VERSION) == 4 %}
+  window.MAPBOX_ACCESS_TOKEN = "{{ config.MAPBOX_ACCESS_TOKEN }}";
+  window.MAPBOX_API_VERSION = "{{ config.MAPBOX_API_VERSION }}";
```

Comment:
Either the double quotes need to be removed, or the comparison in the js file has to be non-strict or with a string